lib/simulator-xcode-9.js

```diff
@@ -54,6 +55,8 @@ class SimulatorXcode9 extends SimulatorXcode8 {
    *   client invisible). `false` by default.
    *   - {DevicePreferences} devicePreferences: preferences of the newly created Simulator
    *   device
+   *   - {object} simulatorWindowCenter: object with x,y coordinates to place the Simulator window
```

Comment:
this docstring should be updated now

Reply:
you can, actually, set this property directly to opts.devicePreferences,
_scaleFactor_ is set separately because of legacy reasons